Matrix_Chain_Multiplication/Matrix_Chain_Multiplication.c

```diff
@@ -0,0 +1,99 @@
+                                                                     /*DESCRIPTION*/
+
+/*Let us define the problem as trying determine the order of multiplication M1*M2*M3*...*Mn using parentheses to denote the order.
```

Comment:
Please shorten your problem description. Such a huge comment is not needed as part of code. A Readme file separately can have such great details. The Problem description should be 1 brief paragraph containing 2-3 lines.